tabular/tests/conftest.py

```diff
@@ -130,7 +130,9 @@ def fit_and_validate_dataset(dataset_name,
                                  refit_full=True,
                                  delete_directory=True,
                                  extra_metrics=None,
-                                 expected_model_count=2):
+                                 expected_model_count=2,
+                                 compile_models=False,
+                                 compiler_configs={}):
```

Comment:
Don't use {} as default (mutable), use None and then if compiler_configs is None: compiler_configs = {}

Reply:
Done.